receiver/jaegerreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,22 @@
+receivers:
+  jaeger:
+  jaeger/customname:
+    protocols:
+      thrift-http:
+        endpoint: "example.com:3456"
```

Comment:
nit: given that the host part is ignored do not use it

Reply:
I think it is still worth using in the test to verify parsing (since the parsing code is present).